main/gy521_imu.h

```diff
@@ -79,5 +79,17 @@ void setupIMU(){
     Serial1.print("Z Acceleration: ");
     Serial1.println(sample.accel.z);
     delay(1000);
-  }
+    imuReading imuSample = getIMU();
+    Serial1.print("X Gyro");
```

Comment:
`print` doesn't add any formatting or spaces, so in this current configuration it'll come out as `X Gyro1.234`. If you want `X Gyro: 1.234` you'll have to change this line to `"X Gyro: "`. Same with the rest.

Reply:
Oh yeah, had some trouble with updating the branch so I had to default to old code that I had written which didn't have the space